packages/store/src/storage.ts

```diff
@@ -1390,6 +1467,10 @@ class Storage {
       },
     };
 
+    // Set initial package owner
+    // TODO: Add email of user
+    packageData.maintainers = [{ name: username || '', email: '' }];
```

Comment:
What if `username` is undefined?

Reply:
I changed it to DEFAULT_USER (Anonymous) if undefined